### ISSUE TITLE
[Bug] Problem to send telemetry data Thingsboard 3.2.1

### ISSUE THREAD

**Issue by Easyproto** [thingsboard/thingsboard#4101]
Hi
I'm testing Thingsboard CE inside a cloud server Debian 10. 
I'm using a real device to send HTTP POST telemetry data, but to avoid too many variables of the problem I decided to use postman to execute telemetry post.
To test the message sent I used an account in demo.thingsboard.io with the following message:
**POST https://demo.thingsboard.io/api/v1/CxxJw4QXekWFQNpc5YwL/telemetry**
**with the body 
{
  "temperature":19
}**
All is OK and the device receive data and display it in last telemetry, so message and post command is OK.
After I used the same method to send telemetry data to my server change the Host and the device token.
Nothing append and Last Telemetry remain empty. I tried also to add a dashboard with a gauge to display value but nothing happens.
I decided to add a Default Rule Chain to the device to test messages route using Debug.
I debag all route up to **save timeseries** and the events in this node is
2021-02-13 14:52:17	ioteasy	OUT	DEVICE	c0a3f2a2-6b0d-4d40-915c-b0d6bde307e7	POST_TELEMETRY_REQUEST	Success				
2021-02-13 14:52:17	ioteasy	IN	DEVICE	c0a3f2a2-6b0d-4d40-915c-b0d6bde307e7	POST_TELEMETRY_REQUEST	True				

The events are correct because the last is Success that mean that "save timeseries" complete succefully.
I checked postgres database and I can find the value in the  ts_kv_latest and also in the database the fags of correct message are 1 and the flags of errors are 0 (so all message was received correctly)

Why data isn't displayed in last telemetry or in the dashboard ?
Seems that system doesn't  check database, seems that scheduler is, for some reason, disabled.
All seems to be "freezed" because all widget, for example the widget in the API usage display, are with "hourglass" without values inside (and I think that this is strage due to I have used API a lots....)

Someone have the same problem or answer ?
Thanks
Giacomo

This sis also true because if i check the API Usage display it's always with "hourglass" waiting for data reception....

**Comment by Easyproto** [comment]
Hi All
Some other informations

Before I forgot to give you description of the system:
I'm using Linux Debian 10 with NGINX, Certbot.
I've installe Thingsboard following your instructions.
Java installed is:
openjdk version "1.8.0_275"
OpenJDK Runtime Environment (build 1.8.0_275-8u275-b01-1~deb9u1-b01)
OpenJDK 64-Bit Server VM (build 25.275-b01, mixed mode)
  
I've tried to re-install all but at this time I installed the previous version 3.2.0.
If I send a telemetri data message using HTTP post command the data is stored correctly in database and if I Debug the Rule Chain all is ok up to "save telemetry" that return succes.
So all is correct but nothing is displayed in last telemetri or in the dashboard.
The dashboard of API usage window are still freezed waiting for data.

Seems that there is a sort of disconnection between data stored and data monitoring/visualizing.
No alarm or event are displayed inside devices even if i set them.

An idea ?
Thanks
Giacomo
